File: spec/python/test_type_ternary.py
import unittest

from type_ternary import TypeTernary

class TestTypeTernary(unittest.TestCase):
    def test_type_ternary(self):
        r = TypeTernary.from_file("src/term_strz.bin")

        self.assertEqual(r.dif.value, 0x65)

File: spec/lua/test_type_ternary.lua
local luaunit = require("luaunit")

require("type_ternary")

TestTypeTernary = {}

function TestTypeTernary:test_type_ternary()
    local r = TypeTernary:from_file("src/term_strz.bin")

    luaunit.assertEquals(r.dif.value, 0x65)
end
